### ISSUE TITLE
Insane loss, 0 accuracy and all inputs are the same

### ISSUE THREAD

**Issue by JorianRennen** [keras-team/keras#14630]
Hi,

I'm relatively new to keras and I've done a few projects, But now I got a problem with my model.
My x_train data has the shape: (161058, 150, 1)
My y_train data has the shape: (161058, 18, 1)
The values of this data vary between 3000 and 50000.  
My layers: 
![image](https://user-images.githubusercontent.com/80458086/117120886-2f952280-ad94-11eb-8303-bc21ef4f535b.png)

My code:
````
model = Sequential()
model.add(LSTM(20, return_sequences=True, input_shape=(150, 1)))
model.add(Dropout(0.5))
model.add(LSTM(40, return_sequences=False))
model.add(Dropout(0.5))
model.add(Dense(1, activation='linear'))
model.compile(loss='mse', optimizer='rmsprop', metrics='accuracy')
print(model.summary())
model.fit(x_train, y_train, batch_size=50, epochs=10)
````

The problem is that I get an extremely high loss, 0 accuracy and always the same output out of this model while the input is different. 
The Epochs and the output data are shown below.
![image](https://user-images.githubusercontent.com/80458086/117125731-36269880-ad9a-11eb-8a31-67374221776f.png)

The output:
![image](https://user-images.githubusercontent.com/80458086/117126858-a681e980-ad9b-11eb-8849-2ef1f9e5f5be.png)

It also should be noted that the output is not the right shape, as the shape is (28422, 1), but it should of course be (28422, 18, 1).

Thanks in advance!

**Comment by jvishnuvardhan** [comment]
@JorianRennen I think you can use something like below to match `y_pred` with `y_true`.

```
model = Sequential()
model.add(LSTM(20, return_sequences=True, input_shape=(150, 1)))
model.add(Dropout(0.5))
model.add(LSTM(40, return_sequences=True,))
model.add(Dropout(0.5))
model.add(Dense(8, activation='linear'))
model.compile(loss='mse', optimizer='rmsprop', metrics='accuracy')
print(model.summary())
```
Output is as shown below
```
Model: "sequential_1"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
lstm_2 (LSTM)                (None, 150, 20)           1760      
_________________________________________________________________
dropout_2 (Dropout)          (None, 150, 20)           0         
_________________________________________________________________
lstm_3 (LSTM)                (None, 150, 40)           9760      
_________________________________________________________________
dropout_3 (Dropout)          (None, 150, 40)           0         
_________________________________________________________________
dense_1 (Dense)              (None, 150, 8)            328       
=================================================================
Total params: 11,848
Trainable params: 11,848
Non-trainable params: 0
_________________________________________________________________
None
```

Also, you need to select correct `metric` and `loss` for your problem. Are you trying to develop a regression (looks like this is the case) model or classification model. Please check [this article](https://machinelearningmastery.com/how-to-choose-loss-functions-when-training-deep-learning-neural-networks/) on choosing right `loss` function.

[Here](https://colab.research.google.com/gist/jvishnuvardhan/6b9a5e84be3e866bb6af1ea40aba82ac/untitled.ipynb). Thanks!

I am closing this issue as this was resolved. Please post any further support kind of questions in [TF Forum](https://discuss.tensorflow.org/tags) where there is a big community to support this kind of questions and the response will be faster. Thanks!
